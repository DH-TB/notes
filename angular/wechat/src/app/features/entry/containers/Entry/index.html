<div class="entry-body">
  <router-outlet></router-outlet>
</div>
<div class="entry-footer">
  <div>tabs</div>
</div>
