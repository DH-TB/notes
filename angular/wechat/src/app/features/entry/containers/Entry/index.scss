@import '../../../../styles/variables.scss';

entry{
  display: flex;
  flex-flow: column nowrap;
  width: 100vw;
  height: 100vh;
  overflow: hidden;
  .entry-body{
    flex: 1;
  }
  .entry-footer{
    height: 48px;
    border-top: 1px solid $gray;
  }
}
