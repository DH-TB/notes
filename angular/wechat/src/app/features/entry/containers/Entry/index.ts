import { Component, ViewEncapsulation } from "@angular/core";

@Component({
  selector: 'entry',
  templateUrl: './index.html',
  styleUrls: ['./index.scss'],
  encapsulation: ViewEncapsulation.None
})
export class Entry{

}
