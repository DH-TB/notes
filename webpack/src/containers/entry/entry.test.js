import { mount } from '@vue/test-utils'
import Entry from './index'
test('renders correctly', () => {
  const wrapper = mount(Entry)
  expect(wrapper.element).toMatchSnapshot()
})
