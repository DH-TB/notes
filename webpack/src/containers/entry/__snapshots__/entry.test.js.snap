// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div>
  <div
    class="entry"
  >
    entry abc
  </div>
   
  <img
    src="../../assets/photo.jpg"
  />
</div>
`;
